Automattic/jetpack
# VideoPress: Extract `VideoThumbnail` from `VideoCard`

To improve readability, and maintenance, let's move VideoThumbnail out of VideoCard.

#### Changes proposed in this Pull Request:

* Extract `VideoThumbnail` from` VideoCard`

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Start `Storybook`
* Navigate to `VideoPress/Components/Video Thumbnail`
* You should see it out of `VideoCard`.
